renderdoc/driver/shaders/spirv/spirv_debug.cpp

```diff
@@ -2848,18 +2848,37 @@ void ThreadState::StepNext(ShaderDebugState *state, const rdcarray<ThreadState>
     }
     case Op::Switch:
     {
-      OpSwitch switch_(it);
+      Id selectorId = Id::fromWord(it.word(1));
```

Comment:
You could use `OpSwitch32` here to get the selector and target. As long as you leave a comment indicating that it's identically encoded aside from the cases that's fine and I've done that in other cases. I think that's preferable to having manual decoding here.